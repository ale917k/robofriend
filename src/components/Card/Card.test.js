import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import React from "react";
import Card from "./Card";

describe("Card", () => {
  it("expects to match Snapshot Object", () => {
    const wrapper = shallow(<Card />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
