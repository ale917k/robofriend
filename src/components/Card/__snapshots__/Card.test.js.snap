// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card expects to match Snapshot Object 1`] = `
<div
  className="grow bg-light-green br3 pa3 ma2 dib bw2 shadow-5"
>
  <img
    alt="robots"
    src="https://robohash.org/undefined?200x200"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
